src/qforte/ucc/uccnpqe.py

```diff
@@ -267,6 +275,13 @@ def get_residual_vector(self, trial_amps):
 
         return residuals
 
+    def get_sum_residual_square(self, tamps):
```

Comment:
Does it make more sense to have this as part of `uccpqeabc.py`?

Reply:
I agree with you. In fact, I wanted to do that but I forgot.